amethyst_ui/src/image.rs

```diff
@@ -12,6 +12,29 @@ pub enum UiImage {
     PartialTexture(Handle<Texture>, TextureCoordinates),
     /// An image backed by a Sprite
     Sprite(SpriteRender),
+    /// An Image backed by a 9-sliced texture
+    NineSlice {
+        ///x starting position on the texture
```

Comment:
nit: could you add a space after each `///`, and capitalize the first letter for sentences?

Reply:
Fixed